polaris/evaluate/metrics/docking_metrics.py

```diff
@@ -0,0 +1,67 @@
+# This script includes docking related evaluation metrics.
+
+from typing import Union, List
+
+import numpy as np
+from rdkit.Chem.rdMolAlign import CalcRMS
+import datamol as dm
+
+
+## The following rmsd implementation are modified based on ppsebusters https://github.com/maabuu/posebusters/blob/main/posebusters/modules/rmsd.py
+
+
+def _rmsd(mol_probe: dm.Mol, mol_ref: dm.Mol) -> float:
+    """Calculate RMSD between predicted molecule and closest ground truth molecule.
+       The RMSD is calculated with first conformer of predicted molecule and only consider heavy atoms for RMSD calculation
+
+    Args:
+        mol_probe: Predicted molecule (docked ligand) with exactly one conformer.
+        mol_ref: Ground truth molecule (crystal ligand) with at least one conformer. If multiple conformers are
+            present, the lowest RMSD will be reported.
+
+    Returns:
+        PoseBusters results dictionary.
+    """
+
+    # copy the molecule for modification.
+    mol_probe = dm.copy_mol(mol_probe)
+    mol_ref = dm.copy_mol(mol_ref)
+
+    # remove hydrogen from molecule
+    mol_probe = dm.remove_hs(mol_probe)
+    mol_ref = dm.remove_hs(mol_ref)
+
+    # calculate RMSD
+    try:
+        return CalcRMS(
+            prbMol=mol_probe, refMol=mol_ref, symmetrizeConjugatedTerminalGroups=True, prbId=-1, refId=-1
+        )
+
+    except RuntimeError:
+        pass
+    except ValueError:
+        pass
```

Comment:
**Suggestion:** Could we add a comment explaining in which cases a user has provided a valid prediction to the benchmark, but the RMSD computation still fails? 

Something like: 
```suggestion
    # This can happen if ...
    except RuntimeError:
        pass
    # This can happen if ...
    except ValueError:
        pass
```